src/_locales/de/messages.json

```diff
@@ -300,7 +300,7 @@
         "description": "Intro page paragraph heading."
     },
     "intro_learns_paragraph": {
-        "message": "Anstatt vorgefertigte Listen zu blockender Domains zu verwenden, entdeckt Privacy Badger Tracker aufgrund ihres Verhaltens automatisch.",
+        "message": "Anstatt vorgefertigte Listen zu blockierender Domains zu verwenden, entdeckt Privacy Badger Tracker aufgrund ihres Verhaltens diese automatisch.",
```

Comment:
If this update makes it so that Privacy Badger learns based on YOUR behavior, it's incorrect. Privacy Badger learns based on the domains' behavior.

Reply:
Same issue as with line 173, will undo the change